src/app/thumbnail-slider.js

```diff
@@ -700,6 +785,15 @@ export default class ThumbnailSlider extends EventSubscriber {
             }, 2000);
     }
 
+    setThumbnailsCount(count) {
+        console.log('setThumbnailsCount:', count, 'was:', this.thumbnails.length);
+        if (typeof count !== 'number') {
+            count = 0;
+        }
+        this.thumbnails.length = count;
+        this.thumbnails.fill({'version': 0, 'title': 'unloaded'});
```

Comment:
Given https://github.com/ome/omero-iviewer/pull/207#issuecomment-421028262 is this okay?